### PR TITLE
landing_worker: implement landing worker management command (bug 1914016)

### PR DESCRIPTION
**Pull request by zzzeid**
- create shell command that starts legacy landing worker
- pass repos to worker instance instead of fetching from env
- set target repo when triggering landing job
- initialize treestatus client in worker instance
- update tests
- add TREESTATUS_URL to settings
- rename worker to "hg-landing-worker"
- remove unneeded treestatus subsystem initialization
- raise and catch ConnectionError when initializing legacy worker
- remove unused variable from `job_processing`

### CODE REVIEW

File: src/lando/api/legacy/workers/landing_worker.py [hunk 1]
```diff
@@ -47,14 +44,13 @@
 
 
 @contextmanager
-def job_processing(worker: LandingWorker, job: LandingJob):
```
Comment (cgsheeh) [inline src/lando/api/legacy/workers/landing_worker.py:50]: The signature is updated here but there are no changes to callers, guessing this belongs in a different PR.

File: src/lando/api/legacy/workers/base.py [hunk 2]
```diff
@@ -33,24 +34,24 @@ def PAUSE_KEY(self) -> ConfigurationKey:
         """Return the configuration key that pauses the worker."""
         raise NotImplementedError()
 
-    def __init__(self, sleep_seconds: float = 5, with_ssh: bool = True):
+    def __init__(self, repos, sleep_seconds: float = 5, with_ssh: bool = True):
```
Comment (cgsheeh) [inline src/lando/api/legacy/workers/base.py:37]: Type hint missing.

**Review by cgsheeh — CHANGES_REQUESTED** [review]
What is the "simple wrapper" mentioned in the title?

**Comment by zzzeid** [comment]
> What is the "simple wrapper" mentioned in the title?

The management command is basically it. I'll update the title to be a little more specific about this.

File: src/lando/api/legacy/workers/landing_worker.py (hunk 1 above)
Comment (zzzeid) [reply src/lando/api/legacy/workers/landing_worker.py:50]: I'll add this in the commit message. It's an unused variable that I just removed while I was here.

Comment (zzzeid) [reply src/lando/api/legacy/workers/landing_worker.py:50]: Also forgot to update the caller, which is now done. This was pulled from a larger PR earlier, but I think it makes more sense to include it in this one. In any case, the latest 3 PRs are closely related, the last in the chain being #108

**Review by cgsheeh — APPROVED** [review]
